cttportfolio.css

```diff
@@ -0,0 +1,941 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	font-size: 62.5%;
+	font-family: 'Montserrat', sans-serif;
+	color: rgb(4, 8, 9);
+}
+
+body {
+	margin: 0;
+	background-color: rgb(2, 8, 9);
+}
+
+main>div {
+	margin: 0 auto;
+	max-width: 1300px;
+}
+
+main {
+	margin-top: 200px;
+}
+
+h4,
+h5,
+h6,
+.pp_body h1 {
+	font-weight: 700;
+	font-size: 6.0rem;
+	text-transform: uppercase;
+	-webkit-text-stroke-width: 0.3px;
+	-webkit-text-stroke-color: black;
+	color: rgb(248, 248, 255);
+}
+
+h5 {
+	font-weight: 600;
+	font-size: 4rem;
+}
+
+.am_text,
+.navlinks a,
+.contactmetext>div,
+img {
+	box-shadow: 0px -2px 5px rgba(2, 8, 9, 0.6);
+	border-radius: 7px;
+}
+
+a.icon_hover img,
+a.pf_sub_links h6,
+.navlinks.icon_hover>a:hover {
+	border: none;
+}
+
+a.icon_hover:hover img,
+a.pf_sub_links:hover h6,
+.navlinks.icon_hover>a:hover {
```

Comment:
this line is identical to line 52, and adds a border to something you previously took a border off of?